src/basic/mountpoint-util.c

```diff
@@ -119,6 +120,28 @@ int name_to_handle_at_loop(
         }
 }
 
+int name_to_handle_at_try_fid(
+                int fd,
+                const char *path,
+                struct file_handle **ret_handle,
+                int *ret_mnt_id,
+                int flags) {
```

Comment:
This is against our usual order of params?

Reply:
it follows the underyling syscall order of params closely.

Reply:
I think we should prefer our coding style here (also for name_to_handle_at_loop). We have `fputs`/... helpers with changed param orders, and the way we handle `path` param is different from the underlying call. Given there would always be differences, I don't think there's too much of a value to follow the underlying func here.

Reply:
it's actually still pretty close. I am fine with changing this (but then also name_to_handle_at_loop() at the same time). But please, let's do that in a separate later PR, it#s already more than large enough.